### ISSUE TITLE
Meteor crashes when changing project source code while Karma is up

### ISSUE THREAD

**Issue by lscordilis** [xolvio/meteor-jasmine#276]
I've noticed since I started writing client-side tests, whenever I make changes to the project source code, i.e. everything in not in the ```tests``` directory, Meteor consistently crashes with this error:

```
C:\Users\lscordilis\AppData\Local\.meteor\packages\meteor-tool\1.1.4\mt-os.windows.x86_32\dev_bundle\lib\node_modules\fibers\future.js:245 
                                               throw(ex);
```
 
Which is consistent with Meteor running out of memory based on my research. This kind of error was very rare before I started using Karma and now it happens almost all the time. I'm not sure what the direct cause of it is but I thought you should know. 

Again, I'm doing all of this on Windows... which I'm sure doesn't help. Also you should know I haven't done anything weird to the Karma config or written any intensive tests, as I haven't been able to progress in my client-side testing due to this and other issues.

**Comment by lscordilis** [comment]
Restarting Windows did not make this problem disappear, as it did for some of the other ones I was encountering yesterday.

**Comment by hunius** [comment]
Same problem here on Windows 10.

**Comment by Sanjo** [comment]
Can you please post the relevant code snippet from `C:\Users\lscordilis\AppData\Local\.meteor\packages\meteor-tool\1.1.4\mt-os.windows.x86_32\dev_bundle\lib\node_modules\fibers\future.js:245` where the error is thrown (probably doesn't help though).

I unfortunately don't have any clue right now why this could happen on Windows.

**Comment by hunius** [comment]
This is the part where the error is thrown. Line 245 is within the catch block.
Hope this helps. Let me know if you need any further information.
Thanks for your help!

```
/**
	 * Mark this future as returned. All pending callbacks will be invoked immediately.
	 */
	"return": function(value) {
		if (this.resolved) {
			throw new Error('Future resolved more than once');
		}
		this.value = value;
		this.resolved = true;

		var callbacks = this.callbacks;
		if (callbacks) {
			delete this.callbacks;
			for (var ii = 0; ii < callbacks.length; ++ii) {
				try {
					var ref = callbacks[ii];
					if (ref[1]) {
						ref[1](value);
					} else {
						ref[0](undefined, value);
					}
				} catch(ex) {
					// console.log('Resolve cb threw', String(ex.stack || ex.message || ex));
					process.nextTick(function() {
						throw(ex);
					});
				}
			}
		}
	},
```

**Comment by Sanjo** [comment]
Should be fixed in sanjo:jasmine@0.19.2. Please let me know if you still encounter errors on Windows. I have just tested it very briefly on Windows.
